.ado/jobs/node-tests.yml

```diff
@@ -31,5 +31,5 @@ jobs:
           inputs:
             versionSpec: '${{ nodeVersion }}.x'
 
-        - script: yarn test
+        - script: yarn test --concurrency 1
```

Comment:
Other options for `yarn test` are provided via `jest` config files. Consider keeping options in a single place. Though ideally, unit tests should run concurrently.

In `*.e2etest.ts` files in `e2e2test` directories of most projects will automatically run without concurrency, in most projects, and will also opt the tesst out of being run automatically by VS Code. As an example, it is where we run live tests of react-native-platform-override against GitHub service. If a test is expensive, it should potentially be moved there, but the UT timeout of 5 seconds should already allow for relatively long pauses for functions that look inexpensive.

Reply:
Here are some good links, to explore how the system works today:
1. https://github.com/microsoft/react-native-windows/blob/main/packages/%40rnw-scripts/just-task/just-task.js
1. https://github.com/microsoft/react-native-windows/blob/main/packages/%40rnw-scripts/jest-unittest-config/jest.unittest.config.js
1. https://github.com/microsoft/react-native-windows/blob/main/packages/%40rnw-scripts/jest-e2e-config/jest.e2e.config.js

Reply:
Notably, the Jest unit test configuration already disables concurrency. The concurrency happens a level higher, in https://github.com/microsoft/react-native-windows/blob/main/lage.config.js.

Lage will run multiple instances of `yarn test` at once, where the package dependency graph lets it. It is not ideal for fine-grained control of test concurrency. The "right way" to do this in Jest configuration over the whole repo, and use its version of workspaces, but it has been finicky before when I tried getting it to work with our repo. It's potentially worth trying again though.